handover_api/utils.py

```diff
@@ -31,9 +32,10 @@ def send_draft(draft):
     message = generate_message(sender.email, receiver.email, subject, template_name, context)
     message.send()
 
+
 def get_accept_url(handover):
-    # TODO: lookup the accept url
-    return 'https://itlab-1.gcb.duke.edu/accept?token=' + str(handover.token)
+    return reverse('accept-process') + "?token=" + str(handover.token)
```

Comment:
I think reverse just returns a relative URL - we need the protocol and host too.